docs/contributing.md

```diff
@@ -49,21 +49,25 @@ source env/bin/activate
 # 3. Install pydantic, dependencies, test dependencies and doc dependencies
 make install
 
-# 4. Checkout a new branch and make your changes
+# 4. Install pre-commit hooks
+pip install pre-commit
```

Comment:
Like I said I think pipx is good for this sort of thing. Maybe add that as a suggestion?